en/docs/install-and-setup/upgrading-wso2-api-manager/260-to-410/upgrading-from-260-to-410.md

```diff
@@ -0,0 +1,1304 @@
+
+    ```
+    [database.user]
+    type = "mysql"
+    url = "jdbc:mysql://localhost:3306/um_db"
+    username = "username"
+    password = "password"
+    ```
+
+    !!! note
+        If you have configured WSO2CONFIG_DB in the previous API-M version, add a new entry to the `<API-M_4.1.0_HOME>/repository/conf/deployment.toml` as below.
+
+        ```
+        [database.config]
+        type = "mysql"
+        url = "jdbc:mysql://localhost:3306/config_db"
+        username = "username"
+        password = "password"
+        ```
+
+    !!! attention "If you are using another DB type"
+        If you are using another DB type other than **H2** or **MySQL** or **Oracle**, when defining the DB related configurations in the `deployment.toml` file, you need to add the `driver` and `validationQuery` parameters additionally as given below.
+
+        ```tab="MSSQL"
+        [database.apim_db]
+        type = "mssql"
+        url = "jdbc:sqlserver://localhost:1433;databaseName=mig_am_db;SendStringParametersAsUnicode=false"
+        username = "username"
+        password = "password"
+        driver = "com.microsoft.sqlserver.jdbc.SQLServerDriver"
+        validationQuery = "SELECT 1"
+        ```
+
+        ```tab="PostgreSQL"
+        [database.apim_db]
+        type = "postgre"
+        url = "jdbc:postgresql://localhost:5432/mig_am_db"
+        username = "username"
+        password = "password"
+        driver = "org.postgresql.Driver"
+        validationQuery = "SELECT 1"
+        ```
+
+        ```tab="Oracle"
+        [database.apim_db]
+        type = "oracle"
+        url = "jdbc:oracle:thin:@localhost:1521/mig_am_db"
+        username = "username"
+        password = "password"
+        driver = "oracle.jdbc.driver.OracleDriver"
+        validationQuery = "SELECT 1 FROM DUAL"
+        ```
+
+        ```tab="DB2"
+        [database.apim_db]
+        type = "db2"
+        url = "jdbc:db2://localhost:50000/mig_am_db"
+        username = "username"
+        password = "password"
+        driver = "com.ibm.db2.jcc.DB2Driver"
+        validationQuery = "SELECT 1 FROM SYSIBM.SYSDUMMY1"
+        ```
+
+    !!! info
+        In API-M 4.1.0, you do not need to configure the registry configurations as you did in the `<OLD_API-M_HOME>/repository/conf/registry.xml` file and the user database configurations as you did in in the `<OLD_API-M_HOME>/repository/conf/user-mgt.xml` file, as those configurations have been handled internally.
+
+    !!! note
+        It is not recommended to use default H2 databases other than `WSO2_MB_STORE_DB` in production. Therefore migration of default H2 databases will not be supported since API-M 4.0.0.
+        It is recommended to use the default H2 database for the `WSO2_MB_STORE_DB` database in API-Manager. So do **not** migrate `WSO2_MB_STORE_DB` database from API-M 2.1.0 version to API-M 4.1.0 version, and use the **default H2** `WSO2_MB_STORE_DB` database available in API-M 4.1.0 version.
```

Comment:
This should be from API-M 2.6.0, right?

Reply:
fixed with https://github.com/wso2/docs-apim/pull/5985/commits/e90c76f0415cfec5924cb7352486cf90790fdf7f